castervoice/lib/ccr/python/python.py

```diff
@@ -120,19 +120,19 @@ class Python(MergeRule):
             # going into the command pallet (cs-p) and typing in "insert snippet"
             # then press enter and then you have choices of snippets show up in the drop-down list.
             # you can also make your own snippets.
-        "try [<exception>]":
+        "try <exception>":
             R(Text("try : ") + Pause("10") + Key("enter/2")
```

Comment:
no space here after try. There should be two enters (why a 0.02 second pause??). The cursor should end up on the line below try

Reply:
jason above also needs fixing. Can you do that here?